cubeseed/businessprofile/tests.py

```diff
@@ -1,3 +1,105 @@
-# from django.test import TestCase
+from rest_framework.test import APITestCase
+from django.urls import reverse
+from rest_framework.test import APIClient
+from rest_framework import status
+from rest_framework.authtoken.models import Token
```

Comment:
This is no longer used right?

Reply:
Yhaa i removed it [here](https://github.com/Cubeseed/Backend/tree/issue%2352l)